source/_integrations/notify.command_line.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Command line Notify"
+title: "Command Line Notify"
```

Comment:
The title is synced from the code base, changing it here will be reverted automatically.